services/app/config/settings/base.py

```diff
@@ -163,10 +163,10 @@ def get_env_variable(var_name):
 EMAIL_ACTIVE_FIELD = "email_verified"
 EMAIL_SERVER = "smtp.gmail.com"
 EMAIL_PORT = 587
-EMAIL_ADDRESS = "pawnhall7@gmail.com"
-EMAIL_FROM_ADDRESS = "noreply@pawnhall.com"
```

Comment:
This change I wasn't sure about but it sounds like this may cause friction with some email providers when the reply email is different than the sender.

Reply:
What does putting `"PawnGuild <pawnhall7@gmail.com>" `mean here?